tests/regression/run_chatbot_regression.py

```
@@ -0,0 +1,199 @@
+    try:
+        with request.urlopen(req, timeout=timeout) as resp:
+            raw = resp.read().decode("utf-8")
+            try:
+                return resp.status, json.loads(raw)
+            except json.JSONDecodeError:
+                return resp.status, {"error": "invalid_json_response", "raw": raw}
+    except error.HTTPError as e:
+        raw = e.read().decode("utf-8", errors="ignore")
+        try:
+            return e.code, json.loads(raw)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)